test/test_torch.py

```diff
@@ -13,6 +13,17 @@
 from funsor.torch import REDUCE_OP_TO_TORCH, Einsum, Tensor, align_tensors, torch_stack, torch_tensordot
 
 
+@pytest.mark.parametrize('output_shape', [(), (2,), (3, 2)], ids=str)
+@pytest.mark.parametrize('inputs', [(), ('a',), ('a', 'b'), ('b', 'a', 'c')], ids=str)
+def test_quote(output_shape, inputs):
```

Comment:
Can you add a test like this for `einsum` expressions?  Maybe just add the following to `test/test_cnf.py`:

```py
@pytest.mark.parametrize('equation,plates', EINSUM_EXAMPLES)
@pytest.mark.parametrize('backend', ['torch', 'pyro.ops.einsum.torch_log'])
@pytest.mark.parametrize('einsum_impl', [einsum, naive_plated_einsum])
def test_normalize_einsum(equation, plates, backend, einsum_impl):
    inputs, outputs, sizes, operands, funsor_operands = make_einsum_example(equation)

    with interpretation(reflect):
        expr = einsum_impl(equation, *funsor_operands, backend=backend, plates=plates)

    expected = reinterpret(expr)
    actual = eval(funsor.quote(expr))
    assert_close(actual, expected)
```